python/paddle/fluid/dygraph/jit.py

```diff
@@ -598,6 +598,13 @@ def train(layer, loader, loss_fn, opt):
     # avoid change user given input_spec
     inner_input_spec = None
     if input_spec is not None:
+        for attr_func in dir(layer):
+            static_func = getattr(layer, attr_func, None)
+            if isinstance(static_func,
+                          StaticFunction) and 'forward' != attr_func:
+                raise ValueError(
+                    "If there are static functions other than forward that need to be saved, the input input_spec should be None, but received input_spec's type is %s."
```

Comment:
maybe we can add some flag to distinguish between argument or method names and English word, such as
`in input_spec` -> `in 'input_sepc'`
`than forward that` -> `than 'forward' that`

Reply:
Done, thanks.